src/design_notebooks/2023spring/sk8944.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Week 1:

Onboarding went on pretty smoothly. Got a taste of the environment we'll be using to design our processor.

Attempted and finished Lab 1. Got exposed to CMake Toolchain, and how CML works.
While attempting the quests for Lab 1:
*make* des not work on windows. Would not geenrate a makefile and thus used cmake --build . instead.
When attempted to build on VsCode, ran into some minor issues with the header file missing directory even when specified in the CML using target_env_variables.

Also, Started reading up on using verilator.